zerver/openapi/zulip.yaml

```diff
@@ -1168,8 +1168,18 @@ paths:
                                 the new stream exists (for private streams, only subscribers and
                                 organization administrators will receive this event).
 
+                                This event is also sent when the user gains access to a previously
+                                inaccessible stream, such as when the user is subscribed to a
+                                stream, their role changes, or when a private stream is made public.
+
                                 Note that organization administrators who are not subscribed will
                                 not be able to see content on the stream; just that it exists.
+
+                                **Changes**: Prior to Zulip 8.0 (feature level 192), this event was
+                                incorrectly only sent when a stream was created or a user was
+                                subscribed to a private stream. For example, guests would
+                                incorrectly not receive this event when gaining access to a public
+                                stream by being subscribed.
```

Comment:
I think it would be better to leave out the first sentence here and focus on what didn't happen prior to this feature level change. Because the current version leaves out that, prior to this feature level, these events were sent when private streams were made public.

I suggest:

> **Changes**: Prior to Zulip 8.0 (feature level 192), this event was not sent when guest users gained access to a public stream by being subscribed.

Reply:
Yes, this seems better. Updated.